scan/src/subscriptions/RequestSub.re

```diff
@@ -74,6 +53,31 @@ module Mini = {
     |}
   ];
 
+  module MultiMiniByOracleScriptConfig = [%graphql
+    {|
+      subscription RequestsMiniByOracleScript($id: bigint!, $limit: Int!, $offset: Int!) {
+        requests(
+          where: {oracle_script_id: $id}
+          limit: $limit
+          offset: $offset
+          order_by: {id: desc}
+        ) {
+          id @bsDecoder(fn: "ID.Request.fromJson")
+          requester @bsDecoder(fn: "Address.fromBech32")
+          oracle_script @bsRecord {
```

Comment:
Should it be Camel case?

Reply:
This is old code. Otherwise I will need to change the record definition too. If it needs to change, then it should go to a separate PR.